spec/system/claims/schools/users/copy_view_users_spec.rb

```diff
@@ -1,68 +1,68 @@
 require "rails_helper"
 
 RSpec.describe "Invite and view a users details copy", service: :claims, type: :system do
```

Comment:
```suggestion
RSpec.describe "Claims user invites and views a user", service: :claims, type: :system do
```
Could we make the naming for this spec more similar to other reworked specs? This change would also apply to the file name `spec/system/claims/schools/users/claims_user_invites_and_views_a_user.rb`

Reply:
Adopted the naming convention.